specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2023-10-01-preview/account.json

```diff
@@ -0,0 +1,2681 @@
+        "operationId": "ContactAssignments_ListContactAssignments",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "scope",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "contactGroupDefinition",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "referenceType",
+            "type": "string",
+            "enum": [
+              "Unknown",
+              "CollectionReference",
+              "ContactGroupDefinitionReference",
+              "DatasourceReference",
+              "ScanReference",
+              "AdaptiveScopeReference"
+            ],
+            "x-ms-enum": {
+              "name": "EntityReferenceType",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "query",
+            "name": "$skipToken",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ContactAssignmentList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ContactAssignments_ListContactAssignments": {
+            "$ref": "./examples/ContactAssignments_ListContactAssignments.json"
+          }
+        }
+      }
+    },
+    "/getScopedContactGroupNames": {
+      "get": {
+        "tags": [
+          "ContactAssignment"
+        ],
+        "description": "Gets the distinct list of contact group names for the provided scope.",
+        "operationId": "ContactAssignments_ListScopedContactGroupNames",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "scope",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "referenceType",
+            "type": "string",
+            "enum": [
+              "Unknown",
+              "CollectionReference",
+              "ContactGroupDefinitionReference",
+              "DatasourceReference",
+              "ScanReference",
+              "AdaptiveScopeReference"
+            ],
+            "x-ms-enum": {
+              "name": "EntityReferenceType",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "query",
+            "name": "$skipToken",
```

Comment:
See other comment about skipToken, you can use it internally inside the nextLink